common-app-lib/resources/templates/structure.html

```diff
@@ -14,7 +14,6 @@
     <!-- EUI -->
     <link href="https://cdn.earthdata.nasa.gov/eui/1.1.3/stylesheets/application.css" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,300" rel="stylesheet" type="text/css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```

Comment:
Does this also fix the issue reported by the Mobile-Friendly Test tool?
Page partially loaded
Not all page resources could be loaded. This can affect how Google sees and understands your page. Fix availability problems for any resources that can affect how Google understands your page.

Reply:
No - those were issues with Google analytics. I'll look into that as well, but I think it's a separate issue

Reply:
Mark and I found that those issues are gone in SIT.